File: Leetcode/python/running-sum-of-1d-array.py
# https://leetcode.com/problems/running-sum-of-1d-array/submissions/

def runningSum(nums):
        print(nums)
        running = []
        numSum = 0
        for num in nums:
            print(num)
            numSum += num
            print(numSum)
            running.append(numSum)
        
       
        return running

File: Leetcode/javascript/running-sum-of-1d-array.js
var runningSum = function(nums) {
    console.log(nums)
    var running = []
    var numSum = 0
    for (i = 0; i < nums.length; i++){
        console.log(nums[i])
        numSum += nums[i]
        console.log(numSum)
        running.push(numSum)
    }


    return running 
};